lib/contacts/contact_details.dart

```diff
@@ -49,9 +49,9 @@ class ContactDetails extends HookConsumerWidget {
         context,
         l10n.removeContact,
         l10n.removeContactConfirmation(contact.name),
-        l10n.yesUppercase,
```

Comment:
.toUppercase() does not work for all languages. Translations should handle lower case, capital case, and upper case separately. Please revert these changes.

Reply:
That's odd because within the translation there's no context available. But if it works like that, I will revert it of course.

Reply:
Reverted with [8f347b0](https://github.com/azbuky/kaspium_wallet/pull/76/commits/8f347b03f4a684662bdd5fd831702a7401a7b80b)
, [0a37c7a](https://github.com/azbuky/kaspium_wallet/pull/76/commits/0a37c7a7f427855f88844d46e3d78732524ee955)
 and [13ba46a](https://github.com/azbuky/kaspium_wallet/pull/76/commits/13ba46a13b3a06d744ec93d9fea1287d557dbfbf)